docs/deployment/authentication.md

```diff
@@ -94,6 +94,25 @@ Here's an example configuration from Google's SAML app creation flow.
 
 <div style={{textAlign: 'center'}}><img width="75%" src="/img/saml-mapping-configuration.png"/></div>
 
+## OAuth
+
+### Github
+
+As a pre-requisite for this configuration make sure to have
+[Flagsmith on Flagsmith](../deployment/overview#running-flagsmith-on-flagsmith) set up.
+
+Sign up and create an organization and a project.
+
+To configure OAuth for Github:
+
+- [Create an OAuth Github application](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app)
+- For the Authorization callback URL use: `https://<your flagsmith domain name>/oauth/github`
+- Create the Flagsmith on Flagsmith flag as it shows [here](../deployment/overview#oauth_github).
```

Comment:
We need to mention the API environment variables here still, why were these removed?